src/components/Asidebar/asidebar.css

```diff
@@ -0,0 +1,45 @@
+
+.aside-container {
+    padding: 0;
```

Comment:
Suggestion: define this in the root or body element so it will inherit by default.